patch-Strip-build-path-directories-in-tools-xen-and-xen-ar.patch

```diff
@@ -0,0 +1,64 @@
+From 5fb3493773b5c2f8b03f55ddf79520bb66cdbc45 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Fr=C3=A9d=C3=A9ric=20Pierret=20=28fepitre=29?=
+ <frederic.pierret@qubes-os.org>
+Date: Sun, 8 Nov 2020 16:17:29 +0100
+Subject: [PATCH] Strip build path directories in tools, xen and xen/arch/x86
+
+Ensure to have a realpath for XEN_ROOT else it fails to substitute
+properly pathes in strings sections
+---
+ tools/Rules.mk        | 2 ++
+ xen/Makefile          | 4 +++-
+ xen/arch/x86/Makefile | 1 +
+ 3 files changed, 6 insertions(+), 1 deletion(-)
+
+diff --git a/tools/Rules.mk b/tools/Rules.mk
+index 67747110b8..cecef42d12 100644
+--- a/tools/Rules.mk
++++ b/tools/Rules.mk
+@@ -204,6 +204,8 @@ endif
+ CFLAGS-$(CONFIG_X86_32) += $(call cc-option,$(CC),-mno-tls-direct-seg-refs)
+ CFLAGS += $(CFLAGS-y)
+ 
++CFLAGS += -ffile-prefix-map=$(XEN_ROOT)=.
```

Comment:
This normally is set from packaging scripts, so I guess it's also unsuitable upstream. But again, here it is fine.